rules/suitesparse.json

```diff
@@ -0,0 +1,43 @@
+{
+  "patterns": ["\\bsuitesparse\\b"],
+  "dependencies": [
+    {
+      "packages": ["libsuitesparse-dev"],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "ubuntu"
+        },
+        {
+          "os": "linux",
+          "distribution": "debian"
+        }
+      ]
+    },
+    {
+      "packages": ["suitesparse-devel"],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "centos"
+        },
+        {
+          "os": "linux",
+          "distribution": "fedora"
+        },
```

Comment:
Thanks for the PR. It looks good to me, except for a failing test on the rule schema. I think it's because we don't currently support Fedora, so removing this entry should get the test passing.

Reply:
Ah, oops. Should of caught that. Thanks, I will fix that now.